src/pubsub/handlers/RunTestHandler.js

```diff
@@ -71,7 +71,7 @@ export default class RunTestHandler extends BaseHandler {
 
   async getValidationReport(message) {
     const sdkMode = message.action;
-    process.env.COMMUNICATION_MODE = message.context.communicationMode;
+    process.env.COMMUNICATION_MODE = message.context.communicationMode ? message.context.communicationMode : CONSTANTS.TRANSPORT;
 
```

Comment:
is there anywhere in Test_Runner which changes the communication mode to SDK ? 
reason for asking : UI changes will work as is right?

Reply:
So suites can be executed by three ways
1) UI
2) FCS( using runTest handler)
3) Standalone (from UI or by passing intent in cold launch using runTest handler)

1) For UI it has it's own buttons, if user wants SDK they will choose SDK mode or if they want transport they will choose transport button, so no default's in UI suite execution.
2) From FCS in intent, communication mode will be received, if not communication they Transport will be default.
3)Standalone, for standalone UI, same way as point 1 it is used.
from intent in cold launch, it has to run via runTest handler

So for point 2 and 3 it has to come to runTest handler only to run suites, so if not communication mode is found then it will made as transport and same env variable is being used in testRunner to get the communication mode